### PR TITLE
[docs] update the readme of x86

### PR DESCRIPTION
**Pull request by pengzhendong**

### CODE REVIEW

File: runtime/server/x86/README.md [hunk 1]
```diff
@@ -139,19 +139,33 @@ wav_path=your_test_wav_path
 ## Run the demo in Docker
 
 When you encounter an issue trying to run the demo, we encourage you to run the demo in
-the Docker container. The image contains the latest release, a shell script and
-several waves to run the demo. Just run it as follows:
+the Docker container.
+
+Or build the Dockerfile yourself, and run it by:
```
Comment (robin1001) [inline runtime/server/x86/README.md:144]: You can use our prebuilt docker, (demo)

Or you can build your own docker by (demo)

**Review by robin1001 — CHANGES_REQUESTED** [review]
please see inline.

File: runtime/server/x86/README_CN.md [hunk 2]
```diff
@@ -106,53 +107,65 @@ wav_path=./20210327_unified_transformer_exp_server/BAC009S0764W0121.wav
...
 ``` sh
-docker run --rm -it mobvoiwenet/wenet:v0.1.0 bash
+docker run --rm -it mobvoiwenet/wenet:v0.5.0 bash
 ```
 
-我们也提供了Dockerfile，可以自己构建一个docker镜像，参考`Dockerfile`文件。
+该镜像包含了编译过程中所依赖的所有第三方库、编译好的文件和预训练模型。
+
+预训练模型在 `/home/model` 目录, 可执行程序在 `/home/wenet/runtime/server/x86/build` 目录。
+
```
Comment (placebokkk) [inline runtime/server/x86/README_CN.md:161]: 也可使用只包含server程序 `websocket_server_main` 的体积更小的镜像`wenet:mini`，从外部加载训练好的模型。

**Review by placebokkk — DISMISSED** [review]

File: runtime/server/x86/README.md [hunk 3]
```diff
@@ -139,19 +139,33 @@ wav_path=your_test_wav_path
 ## Run the demo in Docker
 
 When you encounter an issue trying to run the demo, we encourage you to run the demo in
-the Docker container. The image contains the latest release, a shell script and
-several waves to run the demo. Just run it as follows:
+the Docker container.
+
+The image contains the prebuilt code, and the pretrained model. Just run it as follows:
 
 ``` sh
```
Comment (robin1001) [inline runtime/server/x86/README.md:146]: remove it.

**Review by robin1001 — CHANGES_REQUESTED** [review]
please see inline.

**Review by robin1001 — APPROVED** [review]

**Pull request merged**